changelog.txt

```diff
@@ -1,3 +1,15 @@
+Version 1.0.15
+------------
+- Fix for msal#915
+    * Incorrect id_token returned for B2C app with multiple policies
+- Fix for msal#916
+    * WebView calls loadUrl multiple times over lifecycle
+- Fix for msal#921
+    * WebView displays error when connectivity lost
+- Fix for msal#924
```

Comment:
This issue number does not seem correct

Reply:
it should be 904

Reply:
This should be common 793

https://github.com/AzureAD/microsoft-authentication-library-common-for-android/issues/793